DEVELOPING_OPAL/validate/src/test/java/org/opalj/fpcf/fixtures/purity/ReferenceTypes.java

```diff
@@ -64,24 +64,24 @@ public ReferenceTypes(ReferenceTypes other) {
     // Setting static fields is impure
 
     @Pure(value = "Uses final static immutable object",
-            eps = @EP(cf = DependentCalls.class, pk = "ClassImmutability", p = "ImmutableObject"))
+            eps = @EP(cf = DependentCalls.class, pk = "ClassImmutability", p = "TransitivelyImmutableClass"), analyses = {L1PurityAnalysis.class, L2PurityAnalysis.class})
     @Impure(value = "DependentCalls not recognized as immutable",
-            eps = @EP(cf = DependentCalls.class, pk = "ClassImmutability", p = "ImmutableObject"),
+            eps = @EP(cf = DependentCalls.class, pk = "ClassImmutability", p = "TransitivelyImmutableClass"),
             negate = true, analyses = L0PurityAnalysis.class)
     public static DependentCalls getStaticFinalImmutableObj() {
         return staticFinalImmutableObj;
     }
 
     @Pure(value = "Uses effectively final static immutable object", eps = {
-            @EP(cf = DependentCalls.class, pk = "ClassImmutability", p = "ImmutableObject"),
+            @EP(cf = DependentCalls.class, pk = "ClassImmutability", p = "TransitivelyImmutableClass"),
             @EP(cf = ReferenceTypes.class, field = "staticEffectivelyFinalImmutableObj",
```

Comment:
Fields like this should be renamed to use the new terminology to avoid confusion.